### PR TITLE
small changes to coiled/aws examples

### PR DESCRIPTION
**Pull request by rsignell**
Just a few minor changes that were necessary for me to run the examples on coiled.   I needed to run with the latest version of coiled (I did `pip install coiled==1.3.12.dev12`), for more info see https://github.com/coiled/feedback/issues/271

### CODE REVIEW

File: examples/coiled/aws/coiled-add-asarray.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
     c = xp.add(a, b)
     res = c.compute(
         executor=executor,
-        memory="1 GiB",  # must be greater than allowed_mem
+        memory=["1 GiB", "8 GiB"] # memory range
```
Comment (tomwhite) [inline examples/coiled/aws/coiled-add-asarray.py:24]: Looks like there's a missing comma here?

Also might be worth preserving the previous comment too:

```
# memory range, lower value must be at least allowed_mem"
```

**Review by tomwhite — COMMENTED** [review]
Thanks for submitting this @rsignell!

File: examples/coiled/aws/coiled-add-asarray.py (hunk 1 above)
Comment (TomNicholas) [reply examples/coiled/aws/coiled-add-asarray.py:24]: I didn't realise we could submit a range now, that's nice

Comment (tomwhite) [reply examples/coiled/aws/coiled-add-asarray.py:24]: To be clear - the `memory` argument is passed through to the Coiled function, it's not interpreted by Cubed.

**Comment by tomwhite** [comment]
Thanks for the contribution @rsignell

**Pull request merged**